ui/oopsyraidsy/data/malikahs_well.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+[{
+  zoneRegex: /^Malikah\'s Well$/,
```

Comment:
This works with or without, but just fyi you don't have to escape apostrophes in a regular expression.